tests/storage/concurrent_vm_boot/conftest.py

```
@@ -0,0 +1,107 @@
+                    namespace_name=namespace.name,
+                    client=unprivileged_client,
+                    storage_class_name=storage_class_name_scope_module,
+                ): vm_index
+                for vm_index in range(NUM_CONCURRENT_VMS)
+            }
+            for future in as_completed(futures):
+                vm_index = futures[future]
+                try:
+                    vms.append(future.result())
+                except Exception:
+                    LOGGER.exception(f"Failed to create VM index {vm_index}")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)